src/routes/Dashboard/CardList/Card.tsx

```diff
@@ -0,0 +1,59 @@
+import { IncreaseIcon, DecreaseIcon } from 'assets/svgs'
+import { useMemo } from 'react'
+import styles from './cardList.module.scss'
+import { transformNum } from './utils'
```

Comment:
```suggestion
import { useMemo } from 'react'

import { transformNum } from './utils'

import { IncreaseIcon, DecreaseIcon } from 'assets/svgs'
import styles from './cardList.module.scss'
```

리액트 메서드들이랑 라이브러리에서 불러온 메서드들은 맨 위로, 다른 곳에서 불러온 유틸리티는 중간에, 스타일은 맨 아래로 내리는 게 좋을듯합니당

Reply:
네네 import 정리해둘게요!